File: CS/DashboardExport/Form1.cs
using DevExpress.DashboardCommon;
using System;

namespace DashboardExport
{
    public partial class Form1 : DevExpress.XtraEditors.XtraForm
    {
        CustomDashboardExporter customDashboardExporter;
        CustomDashboardExporter CustomDashboardExporter {
            get {
                if(customDashboardExporter == null)
                    customDashboardExporter = new CustomDashboardExporter(dashboardViewer1);
                return customDashboardExporter;
            }
        }
        public Form1() {
            InitializeComponent();
            
        }
        void Form1_Load(object sender, EventArgs e) {
            Dashboard dashboard = new DevExpress.DashboardCommon.Dashboard();
            dashboard.LoadFromXml(@"Data\CustomerSupport.xml");
            dashboard.DataSources[0].Data = 
                new CustomerSupportData(
                    DataLoader.LoadCustomerSupport(), 
                    DataLoader.LoadEmployees()
                    ).CustomerSupport;
            dashboardViewer1.Dashboard = dashboard;
        }
        void button1_Click_1(object sender, EventArgs e) {
            CustomDashboardExporter.ShowPrintPreview(true);
        }
        private void button2_Click(object sender, EventArgs e) {
            CustomDashboardExporter.ShowPrintPreview(false);
        }
    }
}

File: VB/DashboardExport/Form1.vb
Imports System
Imports System.Windows.Forms
Imports DevExpress.DashboardCommon

Namespace DashboardExport

    Public Partial Class Form1
        Inherits Form

        Private dashboardExporterField As DashboardExporter

        Private ReadOnly Property DashboardExporter As DashboardExporter
            Get
                If dashboardExporterField Is Nothing Then dashboardExporterField = New DashboardExporter(dashboardViewer1)
                Return dashboardExporterField
            End Get
        End Property

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs)
            Dim dashboard As Dashboard = New Dashboard()
            dashboard.LoadFromXml(GetType(Form1).Assembly.GetManifestResourceStream("DashboardExport.CustomerSupport.xml"))
            dashboard.DataSources(0).Data = New CustomerSupportData(LoadCustomerSupport(), LoadEmployees()).CustomerSupport
            dashboardViewer1.Dashboard = dashboard
        End Sub

        Private Sub button1_Click_1(ByVal sender As Object, ByVal e As EventArgs)
            DashboardExporter.ShowPrintPreview(True)
        End Sub

        Private Sub button2_Click(ByVal sender As Object, ByVal e As EventArgs)
            DashboardExporter.ShowPrintPreview(False)
        End Sub
    End Class
End Namespace
